src/main/java/io/github/mountainrange/mule/controllers/StoreController.java

```diff
@@ -31,16 +42,125 @@ public void onSetScene() {
 	}
 
 	@FXML
-	private void handleBuyMuleAction(ActionEvent e) {
-		System.out.println("You must construct additional pylons");
+	private void handleBuySellToggle(ActionEvent e) {
+		if (buySellCombo.getValue().equals(new String("Buy"))) {
+			buyNotSell = true;
+		} else if (buySellCombo.getValue().equals(new String("Sell"))) {
+			buyNotSell = false;
+		}
+	}
+
+	@FXML
+	private void handleExchangeFoodAction(ActionEvent e) {
+		Shop shop = mule.getGameManager().getShop();
+		Player player = mule.getGameManager().getCurrentPlayer();
+		if (buyNotSell) {
```

Comment:
This logic is duplicated 3 times, could you move it to a private method or move this logic out of this controller?